mediapipe/tasks/ios/vision/core/BUILD

```diff
@@ -61,11 +61,5 @@ objc_library(
         "//mediapipe/tasks/ios/core:MPPTaskRunner",
         "//third_party/apple_frameworks:UIKit",
         "@com_google_absl//absl/status:statusor",
-        "@ios_opencv//:OpencvFramework",
-    ] + select({
-        "@//third_party:opencv_ios_sim_arm64_source_build": ["@ios_opencv_source//:opencv_xcframework"],
-        "@//third_party:opencv_ios_sim_fat_source_build": ["@ios_opencv_source//:opencv_xcframework"],
-        "@//third_party:opencv_ios_arm64_source_build": ["@ios_opencv_source//:opencv_xcframework"],
-        "//conditions:default": [],
-    }),
+    ],
```

Comment:
Opencv framework must be conditionally selected in framework and test targets. Linking them here caused some select config mismatches. It is also a better practice to link it to the end products since vision task runner target does not need opencv framework to build.